Objects/longobject.c

```diff
@@ -683,7 +681,7 @@ size_t
 PyLong_AsSize_t(PyObject *vv)
 {
     PyLongObject *v;
-    size_t x, prev;
+    uintmax_t x;
```

Comment:
Obvious question: why the change from `size_t` to `uintmax_t` here?

Reply:
Oops, committed it accidentally.